tests/Unit/Handler/PropertyMapperTest.php

```diff
@@ -277,7 +277,9 @@ public function testPublicNotNullableCustomClassThrowsException(): void
         $propertyMapper = new PropertyMapper();
 
         $this->expectException(\RuntimeException::class);
+        // phpcs:disable Generic.Files.LineLength
```

Comment:
Personally I kind of dislike these type of annotations. Perhaps taking three lines for the exception like this PR does for `src/Handler/PropertyMapper.php` without the annotations would be nicer? Or even using an intermediate variable?
```
$message = "Null provided in json where " . ComplexObject::class . "::child doesn't allow null value";
$this->expectExceptionMessage($message);
```

Reply:
fixed